### PR TITLE
Prepare release v4.6.0

### PR DESCRIPTION
**Pull request by llDrLove**
Following the [steps](https://github.com/digitalocean/csi-digitalocean#releasing) for the release process v4.6.0.

Ran the following command from the `main` branch:
```sh
make NEW_VERSION=v4.6.0 bump-version
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ## unreleased
 
+## v4.6.0 - 2023.05.02
+
+* Update CSI driver for Kubernetes 1.27
+  [[GH-486]](https://github.com/digitalocean/csi-digitalocean/pull/486)
+
```
Comment (timoreimann) [inline CHANGELOG.md:7]: Would you mind adding a small entry for #487?

Comment (llDrLove) [reply CHANGELOG.md:9]: Sure! Small entry added.

**Review by timoreimann — APPROVED** [review]
LGTM with one suggestion.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
 
+## v4.6.0 - 2023.05.02
+
+* Update CSI driver for Kubernetes 1.27
+  [[GH-486]](https://github.com/digitalocean/csi-digitalocean/pull/486)
+* Remove pre-stop driver unregistration
+  [[GH-487]](https://github.com/digitalocean/csi-digitalocean/pull/487)
+* Fix snapshot resizing
+  [[GH-490]](https://github.com/digitalocean/csi-digitalocean/pull/490)
+* Fix VolumeSnapshot spec
```
[the remark this answers is not in the document]
Comment (llDrLove) [reply CHANGELOG.md]: Sure, I removed it.

**Pull request merged**